### ISSUE TITLE
"Post related research" should not clone barnstar tags

### ISSUE THREAD

**Issue by dhaffnersr** [publiclab/plots2#404]
## What happened just before the problem occurred

when I posted a new research note it automatically awarded me a barnstar (empirisim) which was awarded to me in a previous post.
## Relevant URLs

https://publiclab.org/notes/dhaffnersr/03-09-2016/effects-on-uv-radiation-of-glass-and-quartz-using-uv-led-and-uv-laser
## PublicLab.org username

(to help reproduce the issue)
## Browser, version, and operating system

For bug reports, fill out the above template; for feature requests, you can delete the template.

**Comment by jywarren** [comment]
Note to self: this is because the "Post related research" clones all tags.
It may screen some tags already (don't remember) but it should definitely
leave out barnstar tags when cloning.

**Comment by ebarry** [comment]
I tested this issue by searching for an activity which had been awarded a barnstar via this link: https://publiclab.org/tag/barnstar:basic (thank you @bronwen9). 

This activity meets the requirements: https://publiclab.org/notes/deeptalaxmibharadwaj/09-02-2019/building-snake-plant-air-remediation-kit-in-bangalore-india

I clicked: "try it now" button, then reviewed which tags appear in the URL: https://publiclab.org/post?tags=activity:20739,remediation,air-quality,formaldehyde,open-air,filters,indoor-air,pick,current,vocs,first-time-poster,plant-air-purifier-prototypes

I observe that while the original note has the tag `barnstar:basic` , that tag is not included in the new post. Neither are any other tags that are specific for the original activity such as the descriptive power tags `category:build` and `difficulty:easy`-- which is perfect! I am going to close this issue. Thanks!
